src/Dangl.SchneidControl/Services/DataLoggingScheduler.cs

```diff
@@ -20,9 +26,36 @@ public Task StartAsync(CancellationToken cancellationToken)
                 while (_isRunning)
                 {
                     lastStart = DateTime.UtcNow;
+                    var schneidControlSettings = _configuration.Get<SchneidControlSettings>();
                     using var scope = _serviceProvider.CreateScope();
                     var dataLoggingService = scope.ServiceProvider.GetRequiredService<IDataLoggingService>();
-                    await dataLoggingService.ReadAndSaveValuesAsync();
+                    var valuesResult = await dataLoggingService.ReadAndSaveValuesAsync();
+                    if (valuesResult.BufferTemperatureTop?.Value > schneidControlSettings?.MainBufferMaximumTemperature &&
```

Comment:
Should be inverted, we want to check if the heating system has failed.

```suggestion
                    if (valuesResult.BufferTemperatureTop?.Value < schneidControlSettings?.MainBufferMinimumTemperature &&
```